sdk/storage/azure-storage-blobs/test/ut/blob_sas_test.cpp

```diff
@@ -10,6 +10,37 @@
 
 namespace Azure { namespace Storage { namespace Test {
 
+  static std::string AppendQP(Azure::Core::Url url, std::string qp)
```

Comment:
We should define this function in `test_base.hpp` and `test_base.cpp` if it's used more than once

Reply:
test_base.hpp is in `storage common`. This is only used in `storage blobs` test. Do you still recommend moving it in there, rather than a new private header within the storage blobs test source?

Reply:
Yes, who's to say we're not going to use it in another package sometime. It's a general purpose function, it should go in common package.